e2e/specs/drug-orders.spec.ts

```diff
@@ -0,0 +1,168 @@
+    await medicationsPage.page.getByText('Inhalation', { exact: true }).click();
+
+    await medicationsPage.page.getByPlaceholder(/frequency/i).clear();
+    await medicationsPage.page.getByPlaceholder(/frequency/i).click();
+    await medicationsPage.page.getByText('Twice daily', { exact: true }).click();
+
+    await medicationsPage.page.getByLabel(/indication/i).clear();
+    await medicationsPage.page.getByLabel(/indication/i).fill('Hypertension');
+  });
+
+  await test.step('And I save the form', async () => {
+    await expect(page.getByRole('button', { name: /save order/i })).toBeVisible();
+    await page.getByRole('button', { name: /save order/i }).click();
+    await expect(page.getByRole('button', { name: /sign and close/i })).toBeVisible();
+    await page.getByRole('button', { name: /sign and close/i }).click();
+  });
+
+  await test.step('Then I should see a success toast notification', async () => {
+    await expect(medicationsPage.page.getByText(/updated aspirin 81mg/i)).toBeVisible();
+  });
+
+  await test.step('And I should see the updated order in the list', async () => {
+    const headerRow = medicationsPage.medicationsTable().locator('thead > tr');
+    const dataRow = medicationsPage.medicationsTable().locator('tbody > tr');
+
+    await expect(headerRow.nth(0)).toContainText(/start date/i);
+    await expect(headerRow.nth(0)).toContainText(/details/i);
+    await expect(dataRow.nth(0)).toContainText(/aspirin 81mg/i);
+    await expect(dataRow.nth(0)).not.toContainText(/1 tablet/i);
+    await expect(dataRow.nth(0)).toContainText(/2 tablet/i);
+    await expect(dataRow.nth(0)).not.toContainText(/oral/i);
+    await expect(dataRow.nth(0)).toContainText(/inhalation/i);
+    await expect(dataRow.nth(0)).not.toContainText(/once daily/i);
+    await expect(dataRow.nth(0)).toContainText(/twice daily/i);
+    await expect(dataRow.nth(0)).not.toContainText(/3 days/i);
+    await expect(dataRow.nth(0)).toContainText(/5 days/i);
+    await expect(dataRow.nth(0)).not.toContainText(/indication headache/i);
+    await expect(dataRow.nth(0)).toContainText(/indication hypertension/i);
+  });
+
+  await test.step('Then if I launch the overflow menu and click on the `Options` button', async () => {
+    await expect(medicationsPage.page.getByRole('button', { name: /options/i }).nth(0)).toBeVisible();
+    await medicationsPage.page
+      .getByRole('button', { name: /options/i })
+      .nth(0)
+      .click();
+  });
+
+  await test.step('And I click on the `Discontinue` button', async () => {
+    await expect(medicationsPage.page.getByRole('menuitem', { name: /discontinue/i })).toBeVisible();
+    await medicationsPage.page.getByRole('menuitem', { name: /discontinue/i }).click();
+  });
+
+  await test.step('And I save the form', async () => {
+    await expect(page.getByRole('button', { name: /sign and close/i })).toBeVisible();
+    await page.getByRole('button', { name: /sign and close/i }).click();
```

Comment:
Assertions aren't necessary in this context because the statement await page.getByRole('button', { name: /sign and close/i }).click(); will inherently fail if the "sign and close" button is absent. This principle holds true for similar instances elsewhere.